File: src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/ServerInfo.kt
@file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
package com.sourcegraph.cody.agent.protocol_generated;

data class ServerInfo(
  val name: String,
  val authenticated: Boolean? = null,
  val codyEnabled: Boolean? = null,
  val codyVersion: String? = null,
  val authStatus: AuthStatus? = null,
)

File: agent/bindings/csharp/ServerInfo.cs
using System.Text.Json.Serialization;

namespace Cody.Core.Agent.Protocol
{
  public class ServerInfo
  {
    [JsonProperty(PropertyName = "name")]
    public string Name { get; set; }
    [JsonProperty(PropertyName = "authenticated")]
    public bool Authenticated { get; set; }
    [JsonProperty(PropertyName = "codyEnabled")]
    public bool CodyEnabled { get; set; }
    [JsonProperty(PropertyName = "codyVersion")]
    public string CodyVersion { get; set; }
    [JsonProperty(PropertyName = "authStatus")]
    public AuthStatus AuthStatus { get; set; }
  }
}
